### PR TITLE
[stable/phabricator] Fix some typos

### PR DESCRIPTION
**Pull request by carrodher**

### CODE REVIEW

**Comment by carrodher** [comment]
Re-reviewed

File: stable/phabricator/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Phabricator
 
-[Phabricator](https://www.phacility.com) is a collection of open source web applications that help software companies build better software. Phabricator is built by developers for developers. Every feature is optimized around developer efficiency for however you like to work. Code Quality starts with effective collaboration between team members.
+[Phabricator](https://www.phacility.com) is a collection of open source web applications that help software companies build better software. Phabricator is built by developers for developers. Every feature is optimized around developer efficiency for however, you like to work. Code Quality starts with an effective collaboration between team members.
```
Comment (sameersbn) [inline stable/phabricator/README.md]: I think the `,` in `for however, you like to work.` should not be added

**Review by sameersbn — DISMISSED** [review]

**Review by javsalgar — APPROVED** [review]
LGTM

**Pull request merged**